src/components/app/details/cIDetails/CIDetails.tsx

```diff
@@ -704,39 +703,42 @@ const SecurityTab = ({ ciPipelineId, artifactId, status, appIdFromParent }: Secu
         )
     }
 
-    if (scanDetailsLoading) {
+    const severityCount: SeverityCount = getSeverityCountFromSummary(
+        scanResultResponse?.result.imageScan?.vulnerability?.summary.severities,
```

Comment:
Do we need null checks on summary and list and can we create a common variable till scanResultResponse?.result.imageScan?.vulnerability